admin_manual/ai/app_context_chat.rst

```diff
@@ -152,18 +159,35 @@ Configuration Options (OCC)
 
    occ config:app:set context_chat indexing_max_jobs_count --value=3 --type=integer
 
+* ``request_timeout`` integer (default: 3000)
+   Request timeout in seconds for all requests made to the backend.
```

Comment:
This could be a bit more explicit.
```suggestion
   Request timeout in seconds for all requests made to the Context chat backend (the external app in AppAPI).
```